app/assets/javascripts/tagging.js

```diff
@@ -80,3 +85,25 @@ function initTagForm(deletion_path, selector) {
   return el;
 
 }
+
+function geocodeStringAndPan(string, onComplete) {
+  var url = "https://maps.googleapis.com/maps/api/geocode/json?address=" + string.split(" ").join("+");
+  var Blurred = $.ajax({
+      async: false,
+      url: url,
+      complete: onComplete(data.responseJSON.results[0].geometry.location),
+  });
+  onComplete = onComplete || function onComplete(geometry) {
+    lat = geometry.lat;
+    lng = geometry.lng;
+    
+    var geo = [lat, lng];
+
+    if (geo.length > 0) {
+      var confirm = confirm("This looks like a location. Is this full description of the location accurate?");
+      if(confirm) {
+      addTag("lat: " + str(geo[0]));
```

Comment:
Ah! you could also do a single `addTag("lat: " + str(geo[0])" + "," + "lng: " + str(geo[1]))` !

Reply:
Awesome!!